jsapp/js/actions/submissions.es6

```diff
@@ -49,6 +52,26 @@ submissionsActions.getSubmissions.listen((options) => {
     });
 });
 
+/**
+ * This gets an array of submission ids
+ * @param {string} assetUid
+ */
+submissionsActions.getSubmissionsIds.listen((assetUid) => {
+  $.ajax({
```

Comment:
I thought I will start implementing this idea from `dataInterface.es6`:

```
 * TODO: Instead of splitting this huge file it could be a good idead to move
 * all the calls from here to appropriate actions and drop this file entirely.
 * And make actions for calls that doesn't have them.
```

Reply:
Basically: there is no real need for us to separate thos ajax calls from the actions that call them. It seems nice and orderly, but adds complexity and bloat. From now on I plan to create new actions in this way.